images.css

```diff
@@ -0,0 +1,24 @@
+@import url(components.css);
+
+.RES-IMG
+{
+    height: auto;
+    width: 70%;
+}
+.RES-IMG-ROUND
+{
+    border-radius: 50%;
+   
+    height: auto;
+    width: 20%;
+    object-fit:cover;
+}
```

Comment:
responsive image means the image changes its size
when the screen size is changed maintaining its original properties.
for this we do - width: 100%; height: auto;
and apply this property to the img tag

Reply:
Done